src/include/nodes/primnodes.h

```diff
@@ -1121,6 +1121,7 @@ typedef struct CoalesceExpr
 	Oid			coalescecollid; /* OID of collation, or InvalidOid if none */
 	List	   *args;			/* the arguments */
 	int			location;		/* token location, or -1 if unknown */
+	bool			tsql_is_null;	/* whether the calling function is T-SQL ISNULL() */
```

Comment:
This might cause a problem in MVU because both dump and restore are done in postgres dialect, i.e., `tsql_is_null` cannot be set during MVU.

To clarify this, would you check whether a dumped SQL contains correct information? e.g., `bin/pg_dump -d jdbc_testdb -t master_dbo.t_3147_1`